templates/invoice/renderer/default-pdf.pdf.twig

```diff
@@ -126,22 +150,33 @@ mpdf-->
                 <td nowrap class="text-nowrap text-right">{{ rate|money(currency) }}</td>
                 <td nowrap class="text-nowrap text-right">{{ duration }}</td>
                 <td nowrap class="last text-nowrap text-right">{{ entry.rate|money(currency) }}</td>
+                {% endblock %}
             </tr>
+            {% endblock %}
         {% endfor %}
+        {% endblock %}
         </tbody>
         <tfoot>
+        {% block items_foot %}
+            {% if model.calculator.tax > 0 %}
+            {% block items_foot_subtotal %}
```

Comment:
```suggestion
            {% block items_footer_subtotal %}
```

Reply:
I would skip the footer changes too - same reason as for the header. :)